keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/SecurityGroup.kt

```diff
@@ -2,13 +2,16 @@ package com.netflix.spinnaker.keel.api.ec2
 
 import com.netflix.spinnaker.keel.api.ExcludedFromDiff
 import com.netflix.spinnaker.keel.api.Moniker
+import com.netflix.spinnaker.keel.api.support.Tag
 
 data class SecurityGroup(
   val moniker: Moniker,
   val location: Location,
   @get:ExcludedFromDiff
-  val description: String?,
-  val inboundRules: Set<SecurityGroupRule> = emptySet()
+  val description: String? = moniker.toString(),
```

Comment:
The description is apparently mandatory now, but I didn't want to make a ton of unrelated changes, so decided to just set a default if not specified.
![image](https://user-images.githubusercontent.com/1323478/104658561-f0718300-5677-11eb-9ed1-81c870436db4.png)